src/main/java/org/fogbowcloud/manager/core/plugins/PluginFactory.java

```diff
@@ -0,0 +1,52 @@
+package org.fogbowcloud.manager.core.plugins;
+
+import java.lang.reflect.Constructor;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+public class PluginFactory {
+
+    private static final Logger LOGGER = Logger.getLogger(PluginFactory.class.getName());
+    private static final int EXIT_ERROR_CODE = 128;
+
+
```

Comment:
@italo-batista why do we need empty constructors?

Reply:
No need here. I got too _addicted_ to empty constructors. Sorry.